changelog/wave/v1.10.4.mdx

```diff
@@ -0,0 +1,13 @@
+---
+title: Wave v1.10.4
+date: 2024-08-06
+tags: [wave]
+---
+
+## Feature updates and improvements
+
+- Remove contentreader and contentreaderfactory classes by @munishchouhan in https://github.com/seqeralabs/wave/pull/584
```

Comment:
Same question about code font (and applies to all files).

Reply:
Fixed in commit